packages/mobile/src/verify/VerificationInterstitialScreen.tsx

```diff
@@ -1,19 +1,45 @@
+import PulsingDot from '@celo/react-components/components/PulsingDot'
 import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
 import * as React from 'react'
 import { withNamespaces, WithNamespaces } from 'react-i18next'
-import { ScrollView, StyleSheet, Text } from 'react-native'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
 import SafeAreaView from 'react-native-safe-area-view'
+import { isE2EEnv } from 'src/config'
 import { Namespaces } from 'src/i18n'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+
+const SCREEN_DUARTION = 5000 // 5s
+
+const AnimatedCircle = () => (
+  <View style={styles.iconContainer}>
+    <PulsingDot color={colors.inactive} circleStartSize={12} animated={!isE2EEnv} />
+  </View>
+)
 
 class VerificationInterstitialScreen extends React.Component<WithNamespaces> {
   static navigationOptions = null
 
+  componentDidMount() {
+    setTimeout(() => {
+      navigate(Screens.VerificationInputScreen)
+    }, SCREEN_DUARTION)
+  }
```

Comment:
Can this screen be unmounted before the timer triggers?

Reply:
I'll clear the timeout just in case, thanks!